### PR TITLE
Fix note regarding installation of Pipenv in Readme

### PR DESCRIPTION
**Pull request by Etone**
- Changed wording to represent current state
- Added link to Pipenv installation instructions

Fix: #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,9 @@ from there. Additionally, [Gradle](https://gradle.org/) is required to actually
 GatorGradle. A complete example configuration of Gradle and GatorGradle is available
 in the [Sample Lab](https://github.com/GatorEducator/gatorgrader-samplelab) repository.
 
-NOTE: GatorGradle will automatically install [Pipenv](pipenv.readthedocs.io),
-but does not add the executable to `$PATH`. This can be done manually if the
-user wishes to use Pipenv for other projects.
+NOTE: GatorGradle will **NOT** automatically install [Pipenv](pipenv.readthedocs.io). Nothing other then GatorGradle will be installed automatically.
```
Comment (Michionlion) [inline README.md]: Quick correction: GatorGrader will be installed automatically.

**Review by Michionlion — CHANGES_REQUESTED** [review]
Small changes must be made to pass Travis (the build errored due to network conditions the first run, so I've rerun it) -- `README.md` must pass `mdl` (see the log [here](https://travis-ci.org/GatorEducator/gatorgradle/builds/437010797#L904)).

Other than that, looks good! Thanks for the PR! :)

**Comment by Etone** [comment]
Sorry for the inconvenience, seems like i had my mdl plugin in vscode disabled. Should be fixed. While at it i fixed the reference to pipenv docs as they always linked to the local file tree.

**Review by Michionlion — APPROVED** [review]

**Comment by Michionlion** [comment]
No problem, thanks for the quick turnaround!

**Pull request merged**